src/structures/interfaces/InteractionResponses.js

```diff
@@ -100,17 +104,20 @@ class InteractionResponses {
   /**
    * Edits the initial reply to this interaction.
    * @see Webhook#editMessage
-   * @param {string|APIMessage|MessageAdditions} content The new content for the message
-   * @param {WebhookEditMessageOptions} [options] The options to provide
+   * @param {string|APIMessage|MessageAdditions|WebhookEditMessageOptions} options The new options for the message
    * @returns {Promise<Message|Object>}
    * @example
    * // Edit the reply to this interaction
    * interaction.editReply('New content')
    *   .then(console.log)
    *   .catch(console.error);
    */
-  editReply(content, options) {
-    return this.webhook.editMessage('@original', content, options);
+  editReply(options) {
+    if (typeof options === 'string' || options instanceof APIMessage) {
+      return this.webhook.editMessage('@original', options);
+    } else {
+      return this.webhook.editMessage('@original', null, options);
```

Comment:
This method only takes options now, same as InteractionWebhook.editMessage does, it should remain a passthrough, simply removing the content parameter.